data/linux-bridge/002-linux-bridge.yaml

```diff
@@ -40,11 +40,12 @@ spec:
             - -ce
             - |
               cni_mount_dir=/opt/cni/bin
+              host_dir={{ .CNIBinDir }}
               components=("bridge" "tuning")
               for component in "${components[@]}"; do
                 if find "${cni_mount_dir}/${component}" &>/dev/null && ! test -L "${cni_mount_dir}/${component}"; then
                   echo "${component} binary found, creating symbolic link ${cni_mount_dir}/cnv-${component}"
-                  ln -sf "${cni_mount_dir}/${component}" "${cni_mount_dir}/cnv-${component}"
+                  ln -sf "${host_dir}/${component}" "${cni_mount_dir}/cnv-${component}"
```

Comment:
as we can see on the `ls` here
https://github.com/kubevirt/cluster-network-addons-operator/pull/1592#discussion_r1281543418
the files that are created are created fine, because it is mounted,
we need to change the pointers only iiuc,

will double check on CRC by tricking it to do both conditions, one at a time